Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -4659,7 +4715,7 @@ final class SafeDIToolTests: XCTestCase {
 
                     @Instantiable
                     extension RootViewController: Instantiable {
-                        public static instantiate() {
```

Comment:
😅 kinda impressed these tests were working despite the obvious syntax errors in the test code